internal/tiltfile/helm.go

```diff
@@ -103,3 +111,37 @@ func unableToFindHelmErrorMessage() error {
 
 	return fmt.Errorf("Unable to find Helm installation. Make sure `%s` is on your $PATH.", binaryName)
 }
+
+func localSubchartDependenciesFromPath(chartPath string) ([]string, error) {
+	requirementsPath := filepath.Join(chartPath, "requirements.yaml")
+	_, err := os.Stat(requirementsPath)
```

Comment:
idiomatically, you'd just check the os.IsNotExist error from ReadFile, rather than doing a separate Stat syscall.